steps/rabbit/logger.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2021 Telefonica Cybersecurity & Cloud Tech SL
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rabbit
+
+import (
+	"log"
+	"os"
+	"path"
+
+	"github.com/Telefonica/golium"
+	"github.com/sirupsen/logrus"
+)
+
+var logger *Logger
+
+// GetLogger returns the logger for rabbit messages in publish/subscribe.
+// If the logger is not created yet, it creates a new instance of Logger.
+func GetLogger() *Logger {
+	if logger != nil {
+		return logger
+	}
+	dir := golium.GetConfig().Log.Directory
+	path := path.Join(dir, "rabbit-pubsub.log")
+	logger, err := NewLogger(path)
+	if err != nil {
+		logrus.Fatalf("Error creating rabbit logger with file: '%s'. %s", path, err)
+	}
+	return logger
+}
+
+// Logger logs the HTTP request and response in a configurable file.
+type Logger struct {
+	log *log.Logger
+}
+
+// NewLogger creates an instance of the logger.
+// It configures the file path where the HTTP request and response are written.
```

Comment:
HTTP request and response?

Reply:
I guess it happens the same for redis:

https://github.com/Telefonica/golium/blob/master/steps/redis/logger.go